js/Common.js

```diff
@@ -1742,8 +1742,23 @@ if (!CRM.vars) CRM.vars = {};
         }
         $(this).parent().toggleClass('collapsed');
         e.preventDefault();
+      })
+
+      // Save the state for sticky accordions
+      .on('click', 'details.crm-expand-sticky', function(e) {
+        // Workaround to run last, otherwise the open attribute is not yet updated
+        setTimeout(() => {
+          CRM.cache.set(this.id, document.getElementById(this.id).hasAttribute('open'));
+        }, 0);
       });
 
+    // Expand sticky accordions
+    Array.from(document.querySelectorAll('.crm-container details.crm-expand-sticky')).forEach((expander) => {
+      if (CRM.cache.get(expander.id)) {
+        document.getElementById(expander.id).setAttribute('open', '');
```

Comment:
This works if the accordion has a default state of `closed` to open it.
But what if the accordion has a default state of open then we want to close it?